src/controllers/post.js

```diff
@@ -123,10 +123,10 @@ module.exports = {
 			if (!post) {
 				return response.status(404).send('Error: Post Not Found');
 			}
-			post.views = Number(post.views) + 1;
+			post.views = parseInt(post.views, 10) + 1;
 			await post.save();
```

Comment:
loll, was this necessary, its actually a number data type from the data model, that's why i used Number(). Well this works too